src/build/args.rs

```diff
@@ -8,6 +8,10 @@ pub struct BuildArgs {
     #[clap(subcommand)]
     pub subcommand: Option<BuildSubcommands>,
 
+    /// Confirm all prompts automatically.
+    #[arg(long = "yes", default_value_t = false)]
+    pub silent: bool,
```

Comment:
Perhaps `silent` is a bit misleading? It sounds similar to the `quiet` option in `cargo`, but that disables log messages instead of influencing prompts (well, `cargo` doesn't have prompts to be fair).

Maybe `confirm_prompts` or something similar could be clearer?